app/controllers/schools/add_participants_controller.rb

```diff
@@ -64,10 +64,12 @@ def step_param(step)
     end
 
     def back_link_path
-      previous_step = add_participant_form.previous_step(current_step)
-      return schools_cohort_participants_path unless previous_step
-
-      { action: :show, step: step_param(previous_step) }
+      if (previous_step = add_participant_form.previous_step(current_step))
+        { action: :show, step: step_param(previous_step) }
+      else
+        participants = User.order(:full_name).is_participant.in_school(@school.id)
```

Comment:
Why the `order`?

Reply:
Because I just copied it from the participants_controller without really looking what I'm copying :P

Reply:
I'll merge this in for now (as it has literally no effect on anything) and remove when resolving with multistep form changes